File: server/Models/Game.cs
namespace CollectorsVault.Server.Models
{
    public class Game : VaultItem
    {
        public string Platform { get; set; } = string.Empty;
        public string Genre { get; set; } = string.Empty;
        public int ReleaseYear { get; set; }
        public string CoverUrl { get; set; } = string.Empty;
        public string Developer { get; set; } = string.Empty;
        public string Publisher { get; set; } = string.Empty;
    }
}

File: client/src/models/Game.ts
export interface GameLookupResult {
    title: string;
    platform: string;
    releaseYear: number;
    genre: string;
    description: string;
    coverUrl: string;
    developer: string;
    publisher: string;
    igdbId?: number;
}

export interface Game {
    title: string;
    platform: string;
    releaseDate: string;
    genre?: string;
    description?: string;
    coverUrl?: string;
    developer?: string;
    publisher?: string;
}
